core/src/test/scala/com/yahoo/maha/core/query/presto/PrestoQueryGeneratorTest.scala

```diff
@@ -1102,4 +1105,49 @@ af0.campaign_id = c1.c1_id
 
   }
 
+  test("generating presto query with COL Function") {
+    val jsonString =
+      s"""{
+                          "cube": "s_stats_minute",
+                          "selectFields": [
+                              {"field": "Advertiser ID"},
+                              {"field": "Device ID"},
+                              {"field": "Test COL Function"},
+                              {"field": "Test Modifiable COL Function"},
+                              {"field": "Impressions"}
+                          ],
+                          "filterExpressions": [
+
+                              {"field": "Advertiser ID", "operator": "=", "value": "12345"},
+                              {"field": "Day", "operator": "between", "from": "$fromDate", "to": "$toDate"},
+                              {"field": "Impressions", "operator": ">", "value": "1608"}
+                          ],
+                         "sortBy": [
+                           { "field": "Impressions", "order": "Desc" },
+                           { "field": "Advertiser ID", "order": "Asc"}
+                         ],
+                         "additionalParameters": {
+                            "AdditionalColumnInfo":
+                            [
+                              {"field": "keyword_id", "value": "123"},
+                              {"field": "keyword", "value": "potatoes"}
```

Comment:
can you remind why we need to pass two lines instead directly use:
`{"field": "keyword_id", "value": "potatoes"}`
?